src/main/java/no/digipost/api/client/ApiServiceImpl.java

```diff
@@ -54,33 +60,51 @@
 public class ApiServiceImpl implements ApiService {
 
 	private static final String ENTRY_POINT = "/";
-	private final WebTarget webResource;
 	private final long brokerId;
-
+	private CloseableHttpClient httpClient;
+	private final String digipostUrl;
+	private final RequestConfig config;
+	private final HttpClientBuilder httpClientBuilder;
 
 	private final Callable<EntryPoint> entryPoint = new Callable<EntryPoint>() {
 		@Override
         public EntryPoint call() throws Exception {
-			Response response = webResource.path(ENTRY_POINT)
-					.request(DIGIPOST_MEDIA_TYPE_V6)
-					.header(X_Digipost_UserId, brokerId)
-					.get();
-			if (response.getStatus() == OK.getStatusCode()) {
-				return response.readEntity(EntryPoint.class);
-			} else {
-				throw new DigipostClientException(response.readEntity(ErrorMessage.class));
+
+			HttpGet httpGet = new HttpGet(digipostUrl + ENTRY_POINT);
+			httpGet.setHeader(HttpHeaders.ACCEPT, DIGIPOST_MEDIA_TYPE_V6);
+
+			try(CloseableHttpResponse execute = send(httpGet)) {
+
+				if (execute.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
+					EntryPoint entryPoint = JAXB.unmarshal(execute.getEntity().getContent(), EntryPoint.class);
```

Comment:
Dette gjelder for så vidt ganske mange steder rundt om kring, men det skjønte du sikkert ;)